e2eTests/scenarios/FederatedCredentials/runtest.ps1

```diff
@@ -44,6 +45,11 @@ Write-Host -ForegroundColor Yellow @'
 
 $errorActionPreference = "Stop"; $ProgressPreference = "SilentlyContinue"; Set-StrictMode -Version 2.0
 
+if ($linux) {
+    Write-Host 'This test is using bcsamples-bingmaps.appsource and should only run once (either Windows or Linux)'
```

Comment:
Wouldn't that mean that this scenario will always be run only on Windows?

Reply:
This scenario runs on whatever bcsamples-bingmaps.appsource is setup to run on, but for federated credentials, there really are no Linux/Windows specific things.